back/app/services/timeline_service.rb

```diff
@@ -24,15 +24,14 @@ def current_phase(project, time = Time.now)
     end
   end
 
-  def current_or_last_ideation_phase(project, time = Time.now)
+  def current_or_last_can_contain_ideas_phase(project, time = Time.now)
```

Comment:
This makes sense, but I wonder why it wasn't configured like this before. Are there potential side effects?

Reply:
The behaviour of the code remains unchanged, but because budgeting is renamed to voting, this code no longer worked. I fixed it by using `can_contain_ideas?` (not a very sexy name, but this is what we're currently "calling" participation contexts that are either ideation or voting).

It's used to determine the input term in models/custom_field.rb. I think the input term functionality could use some refactoring (for example, a helper method that returns the input term for a given project). I think this is out of scope though, so I didn't change this.